File: Java_Programs/Java_Lab_Program/Lab_4/Program_2.java
// Write a program to check whether a number is even or odd.

import java.util.Scanner;

public class Program_2 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter Any Number To Check Number Is Odd Or Even => ");
        int user_number=sc.nextInt();

        if(user_number % 2 == 0 ){
             System.out.println("Number Is Even "+user_number);
        }else{
            System.out.println("Number Is Odd "+user_number);
        }
    }
}

File: C_DSA_Lab/LAB_1/Program_2.c
// 2. Write a program to find factors of a given number. 

#include<stdio.h>

int main(){
    
     int n;

     printf("Enter Any Number To Find Factors : ");
     scanf("%d",&n);

     printf("Factors Is : ");

     for(int i=1; i<=n; i++){
        if(n%i==0){
             printf(" %d ",i);
        }
     }

}